cli/command/service/opts.go

```diff
@@ -457,6 +457,33 @@ func convertExtraHostsToSwarmHosts(extraHosts []string) []string {
 	return hosts
 }
 
+type isolationOpts struct {
+	value  *container.Isolation
+	source string
+}
+
+func (o *isolationOpts) Set(value string) error {
+	o.source = value
+	tv := container.Isolation(value)
+	o.value = &tv
+	return nil
```

Comment:
This never returns an error; is the error to satisfy an interface?

Reply:
Yes, it is the flag.Value interface.

Reply:
Check ✅